src/Stryker.Core/Stryker.Core/ToolHelpers/MsBuildHelper.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+
+namespace Stryker.Core.ToolHelpers
+{
+    public class MsBuildHelper
+    {
+        private readonly IFileSystem _fileSystem;
+        private readonly IEnumerable<string> possibleLocations;
+
+        public MsBuildHelper(IFileSystem fileSystem = null)
+        {
+            _fileSystem = fileSystem ?? new FileSystem();
+            possibleLocations = new List<string>
+            {
+                @"C:\Windows\Microsoft.Net\Framework64\v4.0.30319\MSBuild.exe",
+                @"C:\Windows\Microsoft.Net\Framework64\v3.5\MSBuild.exe",
+                @"C:\Windows\Microsoft.Net\Framework64\v2.0.50727\MSBuild.exe",
+                @"C:\Windows\Microsoft.Net\Framework\v4.0.30319\MSBuild.exe",
+                @"C:\Windows\Microsoft.Net\Framework\v3.5\MSBuild.exe",
+                @"C:\Windows\Microsoft.Net\Framework\v2.0.50727\MSBuild.exe",
+            };
+        }
+
+        public string GetMsBuildPath()
+        {
+            foreach(string possiblePath in possibleLocations)
+            {
+                if (_fileSystem.File.Exists(possiblePath))
+                {
+                    return possiblePath;
```

Comment:
If we start receiving reports of builds failing, we should make sure to check if perhaps the msbuild version used for the targetframework is actually relevant (IE framework 3.5 wants the msbuild from the 3.5 folder) but for now this is fine.